energy-dao/src/common/rewards_wrapper.rs

```diff
@@ -0,0 +1,34 @@
+use super::unique_payments::UniquePayments;
```

Comment:
I used a modified version of the `UniquePayments` struct, both because I removed a few lines of code that were not needed and also in order to keep the contract completely independent from the auto-farm contract.